pkg/cmd/release/create/gitResources.go

```
@@ -0,0 +1,301 @@
+ which reference them. 
+
+bold(COMMANDS)
+ Any any point, you can enter one of the following:
+ - green(?) to access this help screen
+ - green(y) to accept the list of Git resources and proceed with creating the release
+ - green(u) to undo the last edit you made to Git resource Git refs
+ - green(r) to reset all Git resource Git ref edits
+ - A Git resource Git ref override string.
+
+bold(GIT RESOURCE OVERRIDE STRINGS)
+ Git resource override strings must have 2 or 3 components, separated by a : or =
+ The first component is always the step name. The last component is always the target Git ref
+ When specifying 2 components, it is assumed that this is for the primary git resource.
+ When specifying 3 components, the second component is the name of the Git resource
+ You can specify a * for the Git ref which will default to the step-defined default branch
+
+ Examples:
+   bold(Run Script:refs/heads/my-branch)   dim(# sets primary Git resource in the 'Run Script' step to use the 'refs/heads/my-branch' ref)
+   bold(Deploy helm:TemplateValues-1:refs/tags/1.3.2)   dim(# sets the 'TemplateValues-1' Git resource in the 'Deploy helm' step to use the 'refs/tags/1.3.2' ref)
+   bold(Run Script:*)              dim(# sets primary Git resource in the 'Run Script' step to use the step-defined default branch)
+
+dim(---------------------------------------------------------------------)
+`) // note this expects to have prettifyHelp run over it
+
+func BuildGitResourcesBaseline(deploymentProcessTemplate *deployments.DeploymentProcessTemplate) ([]*GitResourceGitRef, error) {
+	result := make([]*GitResourceGitRef, 0, len(deploymentProcessTemplate.GitResources))
+
+	for _, gitResource := range deploymentProcessTemplate.GitResources {
+		result = append(result, &GitResourceGitRef{
+			ActionName:      gitResource.ActionName,
+			GitRef:          gitResource.DefaultBranch,
+			GitResourceName: gitResource.Name,
+		})
+	}
+
+	return result, nil
```

Don't need to return an error